CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## [UNRELEASED] neptune 1.10.1
```

Comment:
Maybe we want to release it immedietely?

Reply:
Let's do it